base/shared/src/main/scala/scalaz/std/Boolean.scala

```diff
@@ -6,4 +6,13 @@ import utils._
 trait BooleanInstances {
   implicit val booleanDebug: Debug[Boolean] = toStringDebug[Boolean]
   implicit val booleanEq: Eq[Boolean]       = universalEq[Boolean]
+
+  /* https://github.com/scalaz/scalaz/pull/1792
+  implicit val booleanOrd: Ord[Boolean] = instanceOf(new OrdClass[Boolean] {
+    def comp(a: Boolean, b: Boolean) = (a, b) match {
+      case (true, false) => GT
+      case (false, true) => LT
+      case _ => EQ
+    }
+  })*/
```

Comment:
How useful is `Ord[Boolean]`? It seems like either choice is arbitrary.

Reply:
Looked at Haskell's instance and figured "Why not?" :D

Reply:
It's useful. Remember some data structures require a total ordering even though it's arbitrary.